backupccl: enable skipped test

spaskob opened the pull request:
The test was flaky and hence skipped for quite sometime.
I wasn't able to fix it in it's current form because of
the complexity of how we enable job control statements
today. This will be revisited once we have a saner
implementation. The folloup is tracked at #40638.

Touches #40638.
Fixes #24136.

Release notes (bug fix): none.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/40639)
<!-- Reviewable:end -->

lucy-zhang commented:
Do you know what this test failure on teamcity is about? It looks like a timeout.

spaskob commented:
it's a test failure due to a race condition. the backup job every now and
then completes before it realizes it is paused.

On Wed, Sep 11, 2019 at 12:15 PM Lucy Zhang <notifications@github.com>
wrote:

> Do you know what this test failure on teamcity is about? It looks like a
> timeout.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/40639?email_source=notifications&email_token=AANL63ZLBSLRWEULAQ5QZZ3QJEKR7A5CNFSM4IVKMSV2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD6PBH4Q#issuecomment-530453490>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANL6326VIDC3TPHXJTRQ33QJEKR7ANCNFSM4IVKMSVQ>
> .
>

spaskob committed:
backupccl: enable skipped backup test

The test was flaky and hence skipped for quite sometime.
I wasn't able to fix it in it's current form because of
the complexity of how we enable job control statements
today. This will be revisited once we have a saner
implementation. The folloup is tracked at #40638.

Touches #40638.
Fixes #24136.

Release notes (bug fix): none.
